command/agent/config.go

```diff
@@ -260,6 +260,12 @@ type ClientConfig struct {
 	// creating allocations with bridge networking mode. This range is local to
 	// the host
 	BridgeNetworkSubnet string `hcl:"bridge_network_subnet"`
+
+	// AutoFetchCNIPlugins
+	AutoFetchCNIPlugins bool `hcl:"auto_fetch_cni_plugins`
+
+	// AutoFetchCNIPluginsURL
+	AutoFetchCNIPluginsURL string `hcl:"auto_fetch_cni_plugins_url`
```

Comment:
Close quotation the hcl tag:

```suggestion
	AutoFetchCNIPluginsURL string `hcl:"auto_fetch_cni_plugins_url"`
```